docs/site/Model.md

```diff
@@ -194,6 +194,25 @@ Additionally, the model decorator is able to build the properties object through
 the information passed in or inferred by the property decorators, so the
 properties key value pair can also be omitted.
 
+To hide a property, you can use the hiddenProperties setting like this:
```

Comment:
Let's a add a new subsection so that an anchor is created for linking directly to this new text.

```markdown
#### Hidden properties

To hide a property, you can use the `hiddenProperties` setting like this:
```

I think it's also important to explain what does it mean "to hide a property": the property is stored in the database, available in JS/TS code, can be set via POST/PUT/PATCH requests, but it is removed from response bodies (`.toJSON()` output).

Reply:
I think it's a very good change